mapApp/static/mapApp/js/icons.js

```diff
@@ -20,18 +20,14 @@ L.ExtraMarkers.Icon.prototype.options.prefix = 'fa';
 // Icon definitions
 var icons = {
   "bikeRedIcon": L.ExtraMarkers.icon({
-    icon: "fa-bicycle",
-    markerColor: getColor("collision"),
-    iconColor: 'black',
+    innerHTML: '<img src="/media/mapApp/collision_teardrop.png" width=\'27px\' height=\'36px\' style=\'margin-top:5px\'>',
```

Comment:
Can we include the two custom teardrop icons in source control somewhere?

Reply:
I think the blogApp folder is excluded from git so that admin can upload images directly to the server, but I couldn't remember any reason why the mapApp folder would need to be. Just removed it from .gitignore but need to test further, ran into lots of little issues with images so want to make sure this works